lib/api/lead.js

```diff
@@ -65,6 +65,13 @@ Lead.prototype = {
   partitions: function() {
     var path = util.createPath('leads', 'partitions.json');
     return this._connection.get(path);
+  },
+
+  mergeLead: function(winningLead, losingLead, mergeInCRM) {
+    var path = util.createPath('leads', winningLead, 'merge.json');
+    path = util.appendParam(path, 'leadId', losingLead);
+
+    return this._connection.postJson(path, { mergeInCRM: (mergeInCRM === true) });
```

Comment:
Hm, I would prefer to have this function to be more generic since the API itself is more generic. The pattern with the rest of the API calls is that optional parameters are passed in as part of a trailing `options` parameter, as [seen here](https://github.com/usermindinc/node-marketo/blob/master/lib/api/list.js#L14-L21).

So from an API perspective, I'd like to see something like this:

```js
mergeLead: function(winningLead, losingLeads, options) {
}
```

Note that the parameter is `losingLeads` for the `leadIds` parameter. Even if the user passes in a single ID, it should still work. So from the API, I would expect something like this:

```js
mergeLead(1, 10); // lead 10 loses
mergeLead(1, [10,15,20]); // leads 10, 15, and 20 loses
mergeLead(1, 10, {mergeInCRM: true});
```

I know it's annoying to do an object for just a single parameter, but it would be consistent with the rest of the API in this library.

Hope that makes sense.